types/src/transaction/authenticator.rs

```diff
@@ -226,7 +226,7 @@ impl TransactionAuthenticator {
                 // address, so we verify both and only fail if the signature fails for either of
                 // them. The legacy approach is to assume the address of the fee payer is signed
                 // over.
-                let mut to_verify = vec![sender, fee_payer_signer];
+                let mut to_verify = vec![sender];
```

Comment:
you don't need  a vector for this anymore?

Reply:
we append to it in line 232 and take the remaining in line 246.